cicecore/cicedyn/infrastructure/comm/mpi/ice_boundary.F90

```diff
@@ -1552,7 +1608,7 @@ subroutine ice_HaloUpdate2DR8(array, halo,                    &
             !*** correct for offsets
             iSrc = iSrc - ioffset
             jSrc = jSrc - joffset
-            if (iSrc == 0) iSrc = nxGlobal
+            if (iSrc < 1       ) iSrc = iSrc + nxGlobal
```

Comment:
It probably doesn't at the moment, but the new implementation is more general in case greater padding or other tripole (or similar) setups with larger offsets cause it to go negative at some point.  I figured it didn't hurt to make the change while I was there and noticed it.